src/hammer_config/config_src.py

```diff
@@ -944,28 +944,22 @@ def combine_meta(config_dict: dict, meta_setting: str) -> dict:
 def load_config_from_paths(config_paths: Iterable[str], strict: bool = False) -> List[dict]:
     """
     Load configuration from paths containing \*.yml and \*.json files.
-    As noted in README.config, .json will take precedence over .yml files.
 
     :param config_paths: Path to \*.yml and \*.json config files.
     :param strict: Set to true to error if the file is not found.
-    :return: A list of configs in increasing order of precedence.
+    :return: A list of configs in order of specification.
     """
-    # Put the .json configs after the .yml configs to make sure .json takes
-    # precedence over .yml.
-    sorted_paths = sorted(config_paths, key=lambda x: x.endswith(".json"))
-
-    return list(map(lambda path: load_config_from_file(path, strict), sorted_paths))
+    return list(map(lambda path: load_config_from_file(path, strict), config_paths))
 
 
 def load_config_from_defaults(path: str, strict: bool = False) -> List[dict]:
     """
     Load the default configuration for a hammer-vlsi tool/library/technology in
-    the given path, which consists of defaults.yml and defaults.json (with
-    defaults.json taking priority).
+    the given path, which consists of defaults.yml and defaults.json.
```

Comment:
Should note here that the .yml takes precedence now. Or switch the order below.

Reply:
I'll switch the order and make a note.

Reply:
Wait, this is already correct. I didn't need to edit this method.

Reply:
I misread this when I thumbs-up'ed- I thought this was a suggestion. They are equal in priority now, correct? What is the order of precedence, left-to-right?

Reply:
The one later in the list has precedence, so right-to-left, I guess?

Reply:
Got it, that makes sense to me. I'd document that.